frontend/src/app/offers/offer-detail/offer-detail.component.ts

```diff
@@ -16,14 +19,32 @@ export class OfferDetailComponent implements OnInit {
   public offer$: Observable<Offer>;
   public djangoRoot = environment.djangoRoot;
   public getJoinViewUrl = this.offersService.getJoinViewUrl;
+  isUserOrgMember$: Observable<boolean>;
+  user$: Observable<User | null>;
 
   constructor(
     private activatedRoute: ActivatedRoute,
+    private authService: AuthService,
     private offersService: OffersService,
   ) { }
 
   ngOnInit() {
-    this.offer$ = this.activatedRoute.params
-    .switchMap(params => this.offersService.getOffer(params.offerId));
+    this.user$ = this.authService.user$;
+
+    this.offer$ = this.offersService.offer$;
+
+    this.activatedRoute.params
+    .switchMap(params => this.offersService.getOffer(params.offerId))
+    .subscribe();
+  
+    this.isUserOrgMember$ = this.offer$
+     .combineLatest(this.user$, (offer, user): boolean => {
```

Comment:
I don't know ng version here, but hopefully it allows lettable operators. In short - it's kind of new rxjs syntax with better tree shaking being on of the benefits. I don't think it matters a lot here, but for the future reference you can (or maybe should) do it like in the example here: https://medium.com/@cartant/as-youve-used-it-combinelatest-isn-t-really-an-operator-it-s-an-observable-creator-bdab609dae58.